Added recipe anav13jclim.yml (#1466)
ESMValGroup/ESMValTool: doc/sphinx/source/recipes/recipe_anav13jclim.rst

* 3D-scatterplot for different regions showing mean vs. linear trend and the
  model variability index (MVI) as a third dimension (color coded)
* Scatterplot for different regions comparing two variable against each other
  (*cSoil* vs. *cVeg*)

In addition, performance metrics are calculated for all variables using the
performance metric diagnostics (see details in :ref:`nml_perfmetrics`).

.. _mvi calculation:

MVI calculation
---------------

The Model variability index (MVI) on a single grid point (calculated in
``carbon_cycle/mvi.ncl`` is defined as

.. math::

   MVI = \left( \frac{s^M}{s^O} - \frac{s^O}{s^M} \right)^2

where :math:`s^M` and :math:`s^O` are the standard deviations of the annual
time series on a single grid point of a climate model :math:`M` and the
reference observation :math:`O`. In order to get a global or regional result,
this index is simple averaged over the respective domain.

In its given form, this equation is prone to small standard deviations close to
zero. For example, values of :math:`s^M = 10^{-5} \mu` and :math:`s^O = 10^{-7}
\mu` (where :math:`\mu` is the mean of :math:`s^O` over all grid cells) results

> in an MVI in the magnitude of :math:`10^4` for this single grid cell even

Reviewer: ```suggestion
in a MVI of the order of :math:`10^4` for this single grid cell even
```

though the two standard deviations are close to zero and negligible compared to
other grid cells. Due to the use of the arithmetic mean, a single high value is
able to distort the overall MVI.

In the original publication, the maximum MVI is in the order of 10 (for the
variable `gpp`). However, a naive application of the MVI definition yields
values over :math:`10^9` for some models. Unfortunately, `Anav et al. (2013)`_

> do not specify an explanation on how to deal with this problem. Nevertheless,

Reviewer: ```suggestion
do not provide an explanation on how to deal with this problem. Nevertheless,
```
